CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+If you'd like to try or contribute to the plugin, you can clone this repo into your `wp-content/plugins` directory, then activate the plugin. You'll get a separate WordPress menu item called Gutenberg.
+
+Be sure to have <a href="https://nodejs.org/en/">Node installed first</a>. Then, in a terminal, type `npm install` to get the dependencies all set up. Once that finishes, you can type `npm run build` to make the plugin functional, or `npm run dev` to keep the plugin building in the background as you work on it.
```

Comment:
Depending on how much depth we want to go into here, I see this step being the main road bump along the way to getting started. We could add a little more detail about navigating to the plugin directory in the terminal. We could reference that the equivalent to terminal in Windows is the Command Prompt. We might even want to go into some detail (or link to references) about how to set up a local WordPress environment, or directories to find a local install for popular tools (default OSX Apache folder, MAMP, Desktop Server).